packages/select/select.scss

```diff
@@ -192,6 +189,8 @@ $select-hover-option-color--inverted: $svaberg;
     &__option {
         @include jkl.text-style("body/large-screen");
         color: unset;
+        display: flex; // avoids wrapping long names "under" the focus arrow
+        align-items: center; // centers the checkmark when focusing an unselected item
```

Comment:
Vil vi dette? For meg føles det riktigere at det er alignet med teksten. Det har vel uansett vært litt snakk om å endre måten dette indikeres på generelt, men dette ser i hvert fall litt feil ut i mine øyne:

![Skjermbilde 2021-09-28 kl  15 52 13](https://user-images.githubusercontent.com/25739615/135100516-58ebf3dc-93d0-4f6f-9033-cb919cca1789.png)
![Skjermbilde 2021-09-28 kl  15 55 13](https://user-images.githubusercontent.com/25739615/135101041-33224c2a-d974-4668-befe-7a679eb46dd4.png)

Reply:
Hmm, jeg så meg kanskje blind på "box model" og tenkte dette ga mening. Men ser at det kan bli litt rart. Blir det noe som dette da?

```
→ Google og utvalgte
  partnere ✓
```

Reply:
Enten det, eller hvis man vil gjøre det enkelt for seg selv, bare aligne til toppen? 

![Skjermbilde 2021-09-29 kl  09 44 11](https://user-images.githubusercontent.com/25739615/135224989-2f066808-1df6-4038-b270-e84a171026cd.png)
![Skjermbilde 2021-09-29 kl  09 44 19](https://user-images.githubusercontent.com/25739615/135224995-6a33680a-d9e8-4923-8754-f600a7139324.png)

Ellers er det sikkert en god idé å snakke med designerne om alternative løsninger med litt mindre "pynt" 😅 Har blitt nevnt noen ganger at det er mange piler og stæsj. Det henger igjen fra da vi hadde litt færre farger å jobbe med, men er nok verdt å utforske igjen 😊

https://user-images.githubusercontent.com/25739615/135225764-3f70ca7a-7939-4a58-9117-64bb90fbf740.mov